Road/notebooks/feature_selected.ipynb

```diff
@@ -0,0 +1,1198 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #3.        lambda x: feature_selected.extract_id(json_string=x, target_id='user_id')
</pre><p>Usually, using apply(lambda .... ) is not recommended in large data </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mawada-sweis/Road-Status/pull/11/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
Thank you for providing this information, I will deal with it.